Makefile

```diff
@@ -3,7 +3,7 @@ ORG             := cnabio
 BINDIR          := $(CURDIR)/bin
 GOFLAGS         :=
 GOBUILDTAGS     := osusergo
-LDFLAGS         := -w -s
+LDFLAGS         :=
```

Comment:
Remove this change.
(In order to debug a Go binary you need to include DWARF symbols, which are stripped with the flags we use by default to build Signy).